packages/nns/src/governance.canister.ts

```diff
@@ -508,7 +554,16 @@ export class GovernanceCanister {
     if (this.hardwareWallet) {
       return this.disburseHardwareWallet({ neuronId, toAccountId, amount });
     }
-    const request = toDisburseNeuronRequest({ neuronId, toAccountId, amount });
+    // TODO: Test that the new way also works for disbursements.
```

Comment:
is this correct? I assumed this PR has with hardware wallet to do but following lines are meant to be use for non hardware wallet features. see above ` if (this.hardwareWallet) {`

Reply:
I had to fix a problem with how the account identifier is parse to make the extra functionality work.

That fix is also used in here. According to Islam it's weird that it was even working before.

But just in case, I added this to check. I haven't been able to check yet because I had the issue with the transaction fee of governance and ledger being different and disburses don't work.

Yesterday I deployed small13 and today I shuold be able to check this.

I will remove the comment and set the PR to ready if everything works.

Reply:
Super, thanks for the explanation